scripts/deduplicate.py

```
@@ -3,94 +3,194 @@
+        "percentage": removed / original_nonempty * 100 if original_nonempty > 0 else 0,
     }
 
 
+def find_wordlists(root: Path) -> List[Path]:
+    """Find all wordlist files recursively, skipping .git and scripts."""
+    skip_dirs = {'.git', 'node_modules', '__pycache__'}
+    wordlists = []
+    for ext in ('*.txt', '*.lst'):
+        for f in root.rglob(ext):
+            if not any(skip in f.parts for skip in skip_dirs):
+                wordlists.append(f)
```

Ruff (0.15.17)

[warning] Use `list.extend` to create a transformed list

(PERF401)